Atomic/backends/_docker.py

```diff
@@ -451,6 +451,12 @@ def uninstall(self, iobject, name=None, **kwargs):
 
         if args.force:
             self.delete_containers_by_image(iobject, force=True)
+        else:
+            containers_by_image = self.get_containers_by_image(iobject)
+            if len(containers_by_image) > 0:
+                containers_active = ", ".join(["- {}".format(i.name) for i in containers_by_image])
+                raise ValueError("These containers containers that are still using the image, delete them or use --force:\n"
```

Comment:
+                raise ValueError("Containers %s are using this image, delete them first or use --force:\n"